dev/ci/ci-wrapper.sh

```diff
@@ -16,14 +16,17 @@ cd "${DIR}/../.." || exit 1
 
 export TIMED=1
 
-# NB: in CI TERM is unset in the environment
-# when TERM is unset, bash sets it to "dumb" as a bash variable (not exported?)
-if { [ -t 1 ] && ! [ "$TERM" = dumb ]; } || [ "$CI" ]
-then color_wanted=1
-else color_wanted=
+# if COQ_CI_COLOR is set (from the environment) keep it intact (even when it's the empty string)'
+if ! [ "${COQ_CI_COLOR+1}" ]; then
```

Comment:
Is this the same as
```suggestion
if [ -n "${COQ_CI_COLOR+1}" ]; then
```
or
```suggestion
if [ ! -z "${COQ_CI_COLOR+1}" ]; then
```
?

(seems fine as-is, I've just not seen it set up this way before)

Reply:
the opposite, it's the same as `[ -z "${COQ_CI_COLOR+1}" ]`
ie if COQ_CI_COLOR is unset `"${COQ_CI_COLOR+1}"` expands to `""` and the branch is taken
if COQ_CI_COLOR is set, `"${COQ_CI_COLOR+1}"` expands to `"1"` and the branch is not taken